jobs/build_health.rb

```diff
@@ -123,7 +124,7 @@ def get_jenkins_build_health(build)
   latest_build = builds_with_status.first
   return {
     name: latest_build['fullDisplayName'],
-    status: latest_build['result'] == 'SUCCESS' ? SUCCESS : FAILED,
+    status: latest_build['result'] == 'SUCCESS' ? SUCCESS : latest_build['result'] == 'UNSTABLE' ? UNSTABLE : FAILED,
```

Comment:
I find nested ternaries to be very difficult to read. What about something like:

```ruby
jenkins_statuses = { 'SUCCESS' => SUCCESS, 'UNSTABLE' => UNSTABLE }
# ...
status: jenkins_statuses[latest_build['result']] || FAILED
```

Reply:
Thanks for the comments :) make sense.
I'll commit changes later when I have more time or feel free to adjust to your wishes! 👍